packages/berlin/src/pages/Event.tsx

```diff
@@ -47,6 +47,15 @@ function Event() {
   const tabNames = ['upcoming', 'past'];
   const [activeTab, setActiveTab] = useState<string>('upcoming');
 
+  // Update the active tab based on the presence of openCycles
+  useEffect(() => {
+    if (!openCycles || openCycles.length === 0) {
+      setActiveTab('past');
+    } else {
+      setActiveTab('upcoming');
+    }
+  }, [openCycles]);
```

Comment:
this can be done in a different way, maybe just a function that returns 'past' | 'upcoming' and the result is passed as a prop